tests/ui/noobaa_ui/test_namespace_store.py

```diff
@@ -0,0 +1,64 @@
+import logging
```

Comment:
Should we move it under  tests/mcg/ui ?

Reply:
tests/mcg path does not exist..

Reply:
@OdedViner we have ui dir under mcg -> tests/manage/mcg/ui

Reply:
done